static/usage/modal/can-dismiss/boolean/react.md

```diff
@@ -14,11 +14,11 @@ import {
 } from '@ionic/react';
 
 function Example() {
-  const modal = useRef(null);
+  const modal = useRef<HTMLIonModalElement>(null);
   const page = useRef(undefined);
 
-  const [canDismiss, setCanDismiss] = useState(false);
-  const [presentingElement, setPresentingElement] = useState(undefined);
+  const [canDismiss, setCanDismiss] = useState<boolean>(false);
```

Comment:
```suggestion
  const [canDismiss, setCanDismiss] = useState(false);
```
Generally, you don't need to explicitly set the type when it's just a primitive and you have an initial value, since TS can detect the type from that.